src/widgets/answers/__tests__/answers-test.ts

```diff
@@ -47,18 +48,20 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/answers/js/
         indexName: 'instant_search',
         searchClient,
       });
+      search.helper = helper;
```

Comment:
Why did we have to do this?

Reply:
createInitOptions gets the state from the InstantSearch instance (before it was undefined), in theory we could also have grabbed it from the helper argument, but that isn't such a big difference (usually they will be the same)